airflow/providers/amazon/aws/transfers/dynamodb_to_s3.py

```diff
@@ -118,11 +125,77 @@ def __init__(
         self.dynamodb_scan_kwargs = dynamodb_scan_kwargs
         self.s3_bucket_name = s3_bucket_name
         self.s3_key_prefix = s3_key_prefix
+        self.export_time = export_time
+        self.export_format = export_format
 
-    def execute(self, context: Context) -> None:
-        hook = DynamoDBHook(aws_conn_id=self.source_aws_conn_id)
-        table = hook.get_conn().Table(self.dynamodb_table_name)
+    @cached_property
+    def hook(self):
+        """Create DynamoDBHook"""
+        return DynamoDBHook(aws_conn_id=self.source_aws_conn_id)
 
+    def execute(self, context: Context) -> None:
+        if self.export_time:
+            self._export_table_to_point_in_time()
+        else:
+            self._export_entire_data()
+
+    def _create_or_get_waiter(self):
+        """Create or get a waiter"""
+        if CUSTOM_WAITER_NAME in self.hook.list_waiters():
+            return self.hook.get_waiter(CUSTOM_WAITER_NAME)
+        else:
+            with open(self.hook.waiter_path, "r+") as config_file:
+                model_config = json.load(config_file)
```

Comment:
@vincbeck is there a better way to create/update the waiters config?

Reply:
Thanks for pointing me in the right direction, much helpful :). Made changes accordingly.